variables/day2/day3/18/2/22 function/arrowfunc2.js

```diff
@@ -0,0 +1,10 @@
+// if only one statement is there, then () and return keyword can be avoided
+
+const division = (a,b) => a/b
+console.log(division(25,5));
+
+const square = num => num*num
```

Comment:
you have written same code for square and multiplication .

Reply:
add comment